addons/print/models/print_printer.py

```diff
@@ -216,14 +208,11 @@ def spool_report(self, docids, report_name, data=None, title=None, copies=1):
         # Generate dictionary with {report type: [(title, *rendered_document*), ...]} for each report type
         documents = defaultdict(list)
         for report in reports:
-            documents[report.report_type].append(
-                (
-                    ("%s %s" % (report.name, str(docids))) if title is None else title,
-                    report._render(
-                        docids, cpcl_data if report.report_type == "qweb-cpcl" else data
-                    )[0],
-                )
-            )
+            doc_title = title or ("%s %s" % (report.name, str(docids)))
+            rendered_doc = report._render(
```

Comment:
Sorry to be awkward, but this is still a bit awkward to read, black doesn't help here.

Could we add `report_data`?

So:

```
report_data = cpcl_data if report.report_type == "qweb-cpcl" else data
rendered_doc = report._render(docids, report_data)[0]
```

Again, just a suggestion. Happy for this to be resolved.